webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py

```diff
@@ -182,3 +180,16 @@ async def test_params_clip_element_outside_of_window_viewport(
             context=top_context["context"],
             clip=ElementOptions(element=element),
         )
+
+
+async def test_params_origin_invalid(bidi_session, top_context):
```

Comment:
Please add a test before for invalid types. The same applies for `format` as well.

```suggestion
async def test_params_origin_invalid_value(bidi_session, top_context):
```